frontend/src/components/activePage/TableInstance/TableInstance.tsx

```diff
@@ -58,6 +64,27 @@ const TableInstance: FC<ITableInstanceProps> = ({ ...props }) => {
     });
   };
 
+  const checked = !!instances.filter(i => selectiveDestroy?.includes(i.id))
+    .length;
+
+  const indeterminate =
+    instances.filter(i => selectiveDestroy?.includes(i.id)).length !==
+      instances.length && checked;
+
+  const selectGroup = () => {
+    if (checked) {
+      if (indeterminate) {
+        const tmp = instances.filter(i => !selectiveDestroy?.includes(i.id));
+        tmp.forEach(i => selectToDestroy!(i.id));
+      } else {
+        const tmp = instances.filter(i => selectiveDestroy?.includes(i.id));
+        tmp.forEach(i => selectToDestroy!(i.id));
+      }
```

Comment:
```suggestion
      instances.filter(i => indeterminate ^ selectiveDestroy?.includes(i.id)).forEach(i => selectToDestroy!(i.id));
```
Could be cleaner if this actually worked...